Makefile

```diff
@@ -0,0 +1,47 @@
+# Adds support per each translation
+
+docs-lang-es: TRANSLATION = es
+docs-linkcheck-lang-es: TRANSLATION = es
+docs-clean-lang-es:	TRANSLATION = es
+
+docs-lang-ja_JP: TRANSLATION = ja_JP
+docs-linkcheck-lang-ja_JP: TRANSLATION = ja_JP
+docs-clean-lang-ja_JP: TRANSLATION = ja_JP
+
+docs-lang-ml_IN: TRANSLATION = ml_IN
+docs-linkcheck-lang-ml_IN: TRANSLATION = ml_IN
+docs-clean-lang-ml_IN: TRANSLATION = ml_IN
+
+docs-lang-pt_BR: TRANSLATION = pt_BR
+docs-linkcheck-lang-pt_BR: TRANSLATION = pt_BR
+docs-clean-lang-pt_BR: TRANSLATION = pt_BR
+
+docs-lang-ru_RU: TRANSLATION = ru_RU
+docs-linkcheck-lang-ru_RU: TRANSLATION = ru_RU
+docs-clean-lang-ru_RU: TRANSLATION = ru_RU
+
+docs-lang-zh_CN: TRANSLATION = zh_CN
+docs-linkcheck-lang-zh_CN: TRANSLATION = zh_CN
+docs-clean-lang-zh_CN: TRANSLATION = zh_CN
+
+# check if the translation language exists
+TRANSLATION = no_value
+.PHONY: verificate-translation
+verificate-translation:
+	@if [ $(TRANSLATION) == no_value ]; then \
```

Comment:
Sorry that I am not expert on Make. What will be the no_value content?

Reply:
I think its being used as a zero-value. Make will parse the file first (thus initially setting `TRANSLATION = no_value`) and then the target is called. If it's a valid target its `TRANSLATION` is overwritten with an actual value.

But is there a reason not to just use `ifndef` like you were before?

Reply:
Hello, @davidkhala and @btl5037 .

As Bret said, I'm using this assignment as a zero value to verify if the translation lang exists.

What happened here is that I tried Bret's suggestion of using "ifndef" but I couldn't make it work as proposed. 

A few things:
First, What Bret suggested was great, but using the LANG variable wasn't possible because Make would interpret the LANG environment variable from the console.
So, if no LANG was passed, it took the default console language.

Second, When I used ifndef, and the variable wasn't set in the very beginning, I got an error altogether. 

This didn't work:
```
docs-lang-es: TRANS = es

docs-lang-%:
    ifndef TRANS
        $(error "Please use a language specific `make` target such as: `docs-lang-<language>` where `<language> is the `i18n` short code for your language)
    endif
```

This worked:
```
TRANS = us
docs-lang-es: TRANS = es

docs-lang-%:
    ifndef TRANS
        $(error "Please use a language specific `make` target such as: `docs-lang-<language>` where `<language> is the `i18n` short code for your language)
    endif
```

After research, I found out that all ifndef are evaluated in a first pass of the makefile.

check this, please:
https://stackoverflow.com/questions/11994029/makefile-ifeq-when-are-they-evaluated

Therefore, the workaround that I found was to do this target with the sh script in it.

Any suggestions?